### DISCUSSION TITLE
The word transitions to the wrong prototype

### DISCUSSION THREAD

**Discussion by github123666** [explosion/spaCy#13386] [Help: Other Questions]
![image](https://github.com/explosion/spaCy/assets/76271031/8f1cb5e9-c7b1-48f4-8f2f-f4418c0dd729)

**Comment by svlandeg** [comment]
Hi!

Please provide a minimal example we can produce, and specify your exact question in the message body.

You can find a related issue and workaround in this issue: https://github.com/explosion/spaCy/issues/11174
